.github/workflows/lock.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
It looks like the [example](https://github.com/dessant/lock-threads#examples) recommends an initial schedule of checking once an hour to go through all past issues; if it only handles a certain number of issues at a time we may want to do the same here and bump it back up to once a day after it's caught up?